target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz5490.dts

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "vr9_avm_fritzxx90.dtsi"
+
+/ {
+	compatible = "avm,fritz5490", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 5490/5491";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x10000000>;
+	};
+};
+
+&eth0 {
+	mtd-mac-address = <&urlader 0xcc>;
+	mtd-mac-address-increment = <1>;
+};
+
+&gswip_mdio {
+	phy0: ethernet-phy@0 {
+		status = "disabled";
+	};
+
+	phy1: ethernet-phy@1 {
+		status = "disabled";
+	};
+
+	phy5: ethernet-phy@5 {
+		reg = <0x05>;
+	};
+
+	phy6: ethernet-phy@6 {
+		reg = <0x06>;
+		reset-gpios = <&gpio 32 GPIO_ACTIVE_LOW>;
+	};
+
+	phy9: ethernet-phy@9 {
+		reg = <0x09>;
+	};
+
+	phy11: ethernet-phy@11 {
+		status = "disabled";
+	};
+
+	phy13: ethernet-phy@13 {
+		status = "disabled";
+	};
+};
+
+&gswip_ports {
+	port@0 {
```

Comment:
Personally, I would remove the fiber part until it works properly.

Reply:
The 5490 port setup differs quite a bit from the other devices, mdio 6 is the fiber port and as far as I read the avm stock source correctly mdio 0 thru 4 are used for the qca8k switch, so should not be touched by gswip mdio and 11 and 13 are not there, so the disablement overwrites the data from the common include.
Maybe this should be done differently, but there are other common parts. Let me know if the better approach is to copy and include the vr9_avm_fritzxx90.dtsi directly into the 5490 definitions.

Reply:
I'm not a fan of overwriting properties in dts. I would move the configuration of ports that differ from the vr9_avm_fritzxx90.dtsi file to vr9_avm_fritz{3/5/7}490.dts.

Reply:
> Personally, I would remove the fiber part until it works properly.

Do you expect it not to work? I intend to test this when I have time. @kestrel1974, it might just be working right?